.github/CODEOWNERS

```diff
@@ -1,15 +1,2 @@
-
-# Build scripts and Jenkins files should be audited by BLT
-# Any changes to source code of corda-api to be reviewd by C5 team leads
-
-Jenkinsfile        @corda/blt
-.ci/**             @corda/blt
-
-gradle/*           @corda/blt
-*.gradle           @corda/blt
-gradle.properties  @corda/corda5-team-leads
-
-*.kt               @corda/corda5-team-leads
-*.java             @corda/corda5-team-leads
-
-CODEOWNERS         @corda/blt @corda/corda5-team-leads
+# Reduced list for the duration of 5.0 code freeze 
+* @mnesbit @driessamyn @ronanbrowne @rick-r3 @fenryka
```

Comment:
Please include me too

Reply:
Odd you are there on branch but PR isn't reflecting 

https://github.com/corda/corda-api/blob/ronanb/ES-356/update-code-owners/.github/CODEOWNERS#L2

Reply:
Never mind there it is now